Ports/AvailablePorts.md

```diff
@@ -31,7 +31,7 @@ Please make sure to keep this list up to date when adding and updating ports. :^
 | [`frotz`](frotz/)              | Frotz                                         |                   | https://gitlab.com/DavidGriffith/frotz                |
 | [`gcc`](gcc/)                  | GNU Compiler Collection                       | 10.2.0            | https://gcc.gnu.org/                                  |
 | [`genemu`](genemu)             | Genesis / MegaDrive Emulator                  |                   | https://github.com/rasky/genemu                       |
-| [`git`](git/)                  | Git                                           | 2.26.0            | https://git-scm.com/                                  |
+| [`git`](git/)                  | Git                                           | 2.31.1            | https://git-scm.com/                                  |
```

Comment:
This says 2.31.1, package.sh says 2.31.0

Reply:
I fixed :) it remains from various attempts